docs/KlassenTjeneste.adoc

```diff
@@ -602,9 +602,9 @@ _Norwegian extension: Not explicitly specified in CPSV-AP._
 | _English name_ | _sector_ 
 | URI | cv:sector 
 | Verdiområde / _Range_ |skos:Concept 
-| Anvendelse / _Usage note_ | Egenskapen brukes til å referere til industri/sektor som den aktuelle tjenesten er relatert til, eller er ment for. En tjeneste kan relateres til flere industrier/sektorer.  
+| Anvendelse / _Usage note_ | Egenskapen brukes til å referere til industri/sektor som den aktuelle tjenesten er relatert til, eller er ment for. En tjeneste KAN relateres til flere industrier/sektorer.  
 
-_This property represents the industry or sector a Service relates to, or is intended for. Note that a single Service may relate to multiple sectors._ 
+_This property represents the industry or sector a Service relates to, or is intended for. Note that a single Service MAY relate to multiple sectors._ 
 | Multiplisitet / _Multiplicity_ | 0..n 
 | Kravnivå / _Requirement level_ | Valgfri / _Optional_ 
 | Merknad 1 / _Note 1_ |De mulige verdiene for egenskapen er beskrevet i EUs kontrollerte vokabular https://op.europa.eu/en/web/eu-vocabularies/concept-scheme/-/resource?uri=http://publications.europa.eu/resource/authority/data-theme[Data theme &#x29C9;, window="_blank", role="ext-link"].
```

Comment:
Verdien SKAL velges fra EUs kontrollerte vokabular.... ? og
The value MUST be chosen from EU's controlled vocabulary...?

Reply:
Kopiert og oversatt fra EU (EUs måte å si dette på). Rettes for å være konsekvent.